data/brands/amenity/fuel.json

```
@@ -3638,6 +3638,16 @@
         "name": "OK"
       }
     },
+    {
+      "displayName": "OK",
```

They would have to have unique display names, eg:
```suggestion
      "displayName": "OK (NL)",
```

---

Thanks!